src/uct/ib/dc/dc_mlx5.c

```diff
@@ -1365,7 +1365,7 @@ static UCS_CLASS_INIT_FUNC(uct_dc_mlx5_iface_t, uct_md_h tl_md, uct_worker_h wor
     ucs_assert(self->tx.num_dci_pools <= UCT_DC_MLX5_IFACE_MAX_DCI_POOLS);
 
     /* allocate dcis arrays and stack */
-    self->tx.dcis = ucs_calloc((self->tx.ndci * self->tx.num_dci_pools +
+    self->tx.dcis = ucs_calloc(((self->tx.ndci * self->tx.num_dci_pools) +
```

Comment:
minor: need to remove extra () like this:
```c
    self->tx.dcis = ucs_calloc((self->tx.ndci * self->tx.num_dci_pools) +
                               UCT_DC_MLX5_KEEPALIVE_NUM_DCIS,
                               sizeof(*self->tx.dcis),
```

Reply:
done